source/docs/magento1/search-engine.md

```diff
@@ -82,6 +90,146 @@ query Search {
 If you are using the default theme or the theme Chocolatine, the search bar
 should now be visible.
 
+<blockquote class="warning">
+⚠️ Common mistake: For magento 2 the elasticsuite does not index cms pages. To do that you need to install <a href="https://github.com/Smile-SA/magento2-module-elasticsuite-cms-search" target="_blank">Smile elasticsuite cms search module</a> on your magento instance. Failing to do so will cause the top search bar to malfunction. If you do not need the CMS Page search you can always override related `SearchBar` Components/GraphQL queries to disable the cms page search functionality as in the following diff:
```

Comment:
I think this is not correct. Either we state the CMS Search Magento module is a requirement then we document it as such (and then by definition, not following the requirements means a dysfunctioning platform), either we consider it as optional and in the case, FC should work properly without it.

Also here, you are modifying a Magento1 page mentioning a Magento2 module, this is very confusing. I know the corresponding page for Magento2 does not exist, it's a good opportunity to create it.